client/src/app/home/home/account-card/account-card.component.html

```diff
@@ -0,0 +1,10 @@
+<mat-card
+  (click)="onCardClick()"
+  [ngClass]="{ selected: account._id === accountService.activeAccount?._id }"
+  class="card"
+>
+  <mat-card-subtitle class="card__title">{{ account.title }}</mat-card-subtitle>
+  <mat-card-title class="card__amount"
+    >{{ 123 }} {{ account.currency.code }}</mat-card-title
```

Comment:
123? mock data?

Reply:
I had some problems implementing, the sum calculation in backend so in order to move on I used mock data here. The plan is to implement the login on backend today.